repos/system_upgrade/common/actors/scan_default_initramfs/libraries/scan_default_initramfs.py

```diff
@@ -0,0 +1,39 @@
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.models import DefaultInitramfsInfo, DefaultSourceBootEntry
+
+
+def scan_default_initramfs():
+    default_boot_entry = next(api.consume(DefaultSourceBootEntry), None)
+    if not default_boot_entry:
+        raise StopActorExecutionError('Actor did not receive default boot entry info.')
+
+    try:
+        initramfs_info = run(['lsinitrd', '-m', default_boot_entry.initramfs_path], split=True)['stdout']
+
+    except CalledProcessError as err:
+        details = {'error': str(err)}
+        msg = 'Failed to list details (lsinitrd) of the default boot entry\'s initramfs.'
+        raise StopActorExecutionError(msg, details=details)
+
+    dracut_modules_lines = iter(initramfs_info)
+
+    for line in dracut_modules_lines:  # Consume everything until `dracut-modules:` is seen
+        line = line.strip()
+        if line == 'dracut modules:':
+            break
+
+    dracut_modules = []
+    for module_line in dracut_modules_lines:
+        module_line = module_line.strip()
+        if module_line.startswith('========'):
+            break
+
+        dracut_modules.append(module_line.strip())
+
+    api.current_logger().debug('Default boot entry\'s initramfs (%s) has the following dracut modules: %s',
+                               default_boot_entry.initramfs_path, dracut_modules)
```

Comment:
I think we don't like this old formatting style

```suggestion
    api.current_logger().debug('Default boot entry\'s initramfs ({}) has the following dracut modules: {}'.format(default_boot_entry.initramfs_path, dracut_modules)
```

Reply:
The difference is that, with the old style, no additional formatting work will be done when the logger has configured a higher logging priority. Using your solution, we always format the string. In the context of leapp , this makes no difference as debug logs are always stored. At least, for me, there is no difference from the maintenance perspective. I will change it to use fstrings -- this way, we can immediately know what is being placed where.

Reply:
Sure, fstrings are even better, I just forgot that we don't need to make code in common repo python2.6 compatible anymore.

Reply:
Well, I cannot use fstrings because of a linter rule. I converted the code to `.format()` 
```
[W1203 logging-fstring-interpolation] Use lazy % or .format() or % formatting in logging functions File: repos/system_upgrade/common/actors/scan_default_initramfs/libraries/scan_default_initramfs.py, line 34, in scan_default_initramfs
```